MVVM/ViewModel/DriveMode_UIOverlay.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Godot;
+using RoverControlApp.Core;
+using RoverControlApp.MVVM.Model;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace RoverControlApp.MVVM.ViewModel;
+
+public partial class DriveMode_UIOverlay : UIOverlay
+{
+	[Export] 
+	PanelContainer _panelContainer;
+
+	private int _inputMode; 
+
+	public override Dictionary<int, Setting> Presets { get; } = new()
+	{
+		{ 0, new(Colors.DarkGray, Colors.LightGray, "Drive: Compatibility", "Drive: ") },
+		{ 1, new(Colors.LightGreen, Colors.DarkGreen, "Drive: Ackermann","Drive: ") },
+		{ 2, new(Colors.LightSalmon, Colors.DarkRed, "Drive: Crab","Drive: ") },
+		{ 3, new(Colors.Yellow, Colors.Black, "Drive: Spinner","Drive: ") },
+		{ 4, new(Colors.DarkBlue, Colors.LightBlue, "Drive: E-Brake","Drive: ") }
+	};
+
+	public Task KinematicModeChangedSubscriber(MqttClasses.KinematicMode newMode)
+	{
+		ControlMode = (int)newMode;
+
+		return Task.CompletedTask;
+	}
+
+	public Task ControlModeChangedSubscriber(MqttClasses.ControlMode newMode)
+	{
+		_inputMode = (int)newMode;
+		ControlMode = ControlMode;
+		UpdateIndicatorVisibility();
+
+		return Task.CompletedTask;
+	}
+
+	public override void _Ready()
+	{
+		base._Ready();
+		ControlMode = 1;
+	}
+
+	void UpdateIndicatorVisibility()
+	{
+		if (_inputMode != 1)
```

Comment:
it not consistent now :C 
see: https://github.com/wisniax/RoverControlApp/commit/5ed280b9da4555ee40b428e3d684a0ecffc34549#diff-4e1d48340991dc6ba190b46e8932ff9d5376a500b2d4c8a4403834b7fde20e20R56

Reply:
Missed this one :c. Adressed in [1236c7d](https://github.com/wisniax/RoverControlApp/pull/36/commits/1236c7d63c6477b17e612d8260f3a4a5416c6848)